draft-quic-atsss-reqs.mkd

```diff
@@ -471,12 +471,28 @@ This document discusses three approaches:
 ### Single QUICv1 Tunnel with Unreliable Datagram Extension and Connection Migration
 
 
-In this case, there is one QUIC session between the UE and the ATSSS proxy.
-This QUIC session uses only one access network at a time. Thanks to the
-connection migration capability of QUIC, it is possible to support the
-switching and steering functions of ATSSS, but splitting cannot be supported. 
-
-This is not an option for ATSSSv2.
+QUIC can be used as a tunneling protocol between the UE and the ATSSS proxy. Use of the Unreliable 
+Datagram Extension avoids unnecessary delays due to local retransmissions and, more important, subsequently 
+head-of-line blocking.
+
+In this case, there is (only) one QUIC connection between the UE and the ATSSS proxy for a given flow.
+And as such, that QUIC connection uses only one access network at a time. 
+However, given the connection migration capability of QUIC, the QUIC connection could be
+moved over to another access network, e.g. when the network indicates that the currently used 
+access network would go away or that its capacity becomes limited. Depending of the current flow of traffic 
+either the UE or the ATSSS proxy could initiate the connection migration to start the path
```

Comment:
Not sure it is accurate that a proxy can initiate a connection migration (the proxy will only have one IP address, btw). 

I guess what is meant here is that the proxy can use another destination @ to reach the UE. This assumes that the proxy knows (by some magic) the @es of the UE. Even with that assumption, this is not a connection migration per se.